soaks/tests/http_pipelines_no_grok_blackhole/vector.toml

```diff
@@ -20,8 +20,14 @@ decoding.codec = "bytes"
 type = "remap"
 inputs = ["logs"]
 source = '''
-., err = parse_json(.message)
-.custom = {}
+if starts_with(strip_whitespace!(.message), "{") {
+  custom, err = parse_json(.message)
+  if (err == null) {
+    .custom, err = compact(custom, string: false)
+  }
+}
+.custom.message = .message
+.tags = []
```

Comment:
I think what you have here is different from before. I think this one can be left as-is?